Fhi.HelseId.Web/Infrastructure/AutomaticTokenManagement/AutomaticTokenManagementConfigureCookieOptions.cs

```diff
@@ -9,9 +8,9 @@ namespace Fhi.HelseId.Web.Infrastructure.AutomaticTokenManagement
 {
     public class AutomaticTokenManagementConfigureCookieOptions : IConfigureNamedOptions<CookieAuthenticationOptions>
     {
-        private readonly AuthenticationScheme _scheme;
+        private readonly AuthenticationScheme? _scheme;
 
-        public AutomaticTokenManagementConfigureCookieOptions(IAuthenticationSchemeProvider provider,ILogger<AutomaticTokenManagementConfigureCookieOptions> logger)
+        public AutomaticTokenManagementConfigureCookieOptions(IAuthenticationSchemeProvider provider, ILogger<AutomaticTokenManagementConfigureCookieOptions> logger)
         {
             logger.LogMember();
             _scheme = provider.GetDefaultSignInSchemeAsync().GetAwaiter().GetResult();
```

Comment:
With the proposed change in this PR we risk starting to fail silently, to handle this we can throw an exception in case _scheme is null here.

Reply:
Now throws exception if scheme = null. This will be logged by the infrastructure as error:

{"Timestamp":"2024-10-07T11:55:47.8629004+02:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","RenderedMessage":"An unhandled exception has occurred while executing the request.","TraceId":"696b3bd26e9a3386633f0ee3fc1f62e4","SpanId":"cdaf192f12532e5a","Exception":"System.InvalidOperationException: Field _scheme cannot be null (...)